march_hardware/src/EtherCAT/EthercatMaster.cpp

```diff
@@ -96,7 +97,7 @@ void EthercatMaster::ethercatSlaveInitiation(std::vector<Joint>& joints)
     {
       ec_slave[joint.getIMotionCubeSlaveIndex()].PO2SOconfig = setSlaveWatchdogTimer;
     }
-    joint.initialize(this->cycle_time_ms_);
+    reset |= joint.initialize(this->cycle_time_ms_);
```

Comment:
```suggestion
    reset = joint.initialize(this->cycle_time_ms_);
```
Why not this?

Reply:
This has already been implemented as of writing this reply